data/events.yml

```diff
@@ -1,3 +1,67 @@
+- lang: it
+  startDate: '2022-11-30'
+  endDate: '2022-11-30'
+  online: true
+  speaker: Francesco Stanieri
+  title: Meetup HDG Rome 12
+  subject: 'Come creare un servizio di chat Serverless'
+  url: https://www.meetup.com/hdg-rome/events/289209400/
+- lang: en
+  startDate: '2022-12-02'
+  endDate: '2022-12-02'
+  location: Ultrecht, The Netherlans
+  speaker: Julien Lengrand-Lambert
+  title: 'Flock. Meetups'
+  subject: 'Introducing Kotlin In Your Organization, From The Ground Up'
+  url: https://www.meetup.com/flock-meetups/events/289393375/
+- lang: en
+  startDate: '2022-11-11'
+  endDate: '2022-11-11'
+  online: true
+  speaker: Pamela Hill
+  title: Women Who Code
+  subject: 'Kotlin Multiplatform Mobile for Skeptics'
+  url: https://www.meetup.com/nyandroiddevelopers/events/289399273/
+- lang: en
+  startDate: '2022-12-10'
+  endDate: '2022-12-10'
+  online: true
+  speaker: Amanjeet Singh
+  title: Chicago Kotlin User Group
+  subject: 'Locking Horns with Native Crashes'
+  url: https://www.meetup.com/chicago-kotlin/events/289608325/
+- lang: en
+  startDate: '2022-12-08'
+  endDate: '2022-12-08'
+  online: true
+  speaker: To be announced
+  title: Monthly Kotlin
```

Comment:
@MKrishtal let's add 'San Diego Kotlin User Group' here and 'Monthly Kotlin' to the subject. Otherwise it looks a bit confusing

Reply:
Done